src/transformers/models/auto/auto_factory.py

```diff
@@ -517,6 +542,36 @@ def register(cls, config_class, model_class, exist_ok=False):
             )
         cls._model_mapping.register(config_class, model_class, exist_ok=exist_ok)
 
+    @classmethod
+    def _check_and_return_if_adapter_model(
```

Comment:
See comment above on the name

Reply:
fixed in cd22c543ef06b336dc5735a9fdc149c87c20d9f3